src/Microsoft.AspNet.OData.Shared/Formatter/ODataOutputFormatterHelper.cs

```diff
@@ -203,6 +203,7 @@ internal static bool TryGetCharSet(MediaTypeHeaderValue mediaType, IEnumerable<s
                 writeContext.Path = path;
                 writeContext.MetadataLevel = metadataLevel;
                 writeContext.QueryOptions = internalRequest.Context.QueryOptions;
+                writeContext.IsUntyped = typeof(IEdmObject).IsAssignableFrom(type);
```

Comment:
>writeContext.IsUntyped = typeof(IEdmObject).IsAssignableFrom(type); [](start = 16, length = 67)

why do you have such line code?

IsUntyped is calculated in the ODataSerializerContext